config/producer.properties

```diff
@@ -18,36 +18,28 @@
 
 # list of brokers used for bootstrapping knowledge about the rest of the cluster
 # format: host1:port1,host2:port2 ...
-metadata.broker.list=localhost:9092
+bootstrap.servers=localhost:9092
+
+# specify the compression codec for all data generated: none, gzip, snappy, lz4
+compression.type=none
 
 # name of the partitioner class for partitioning events; default partition spreads data randomly
 #partitioner.class=
 
-# specifies whether the messages are sent asynchronously (async) or synchronously (sync)
-producer.type=sync
-
-# specify the compression codec for all data generated: none, gzip, snappy, lz4.
-# the old config values work as well: 0, 1, 2, 3 for none, gzip, snappy, lz4, respectively
-compression.codec=none
-
-# message encoder
-serializer.class=kafka.serializer.DefaultEncoder
```

Comment:
There are key.serializer and value.serializer but they're also constructor parameters, so I wasn't sure if it was useful to have them here. You think we should have them here?